engine/graphics/src/graphics.cpp

```diff
@@ -75,12 +75,12 @@ namespace dmGraphics
     static bool SelectAdapterByPriority()
     {
         GraphicsAdapter* next     = g_adapter_list;
-        GraphicsAdapter* selected = next;
```

Comment:
In case if first graphics adapter is not supported by platform but has highest priority - it was selected.
Current case:
1. First adapter is WebGPU.
2. WebGPU has priority 0, OpenGL - 1
3. WebGPU isn't supported by browser.
4. OpenGL can't be selected because it has lower priority.
5. As result - error at runtime.